### PR TITLE
Fix: FreeTube Subscription Import

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
This PR fixes importing FreeTube subscriptions for people with more than one profile.

### CODE REVIEW

**Comment by Bnyro** [comment]
This assumes there's one json "file" per line, right?
I think this could break the import feature for some users because not everyone's aware they need to keep the json per profile in one line.
At FreeTube, do you import the subscriptions line per line as well?

**Comment by Bnyro** [comment]
I think that it'd make sense to do it like that if multiple different profiles are contained in the file,
however it'd probably also make sense to have some sort of fallback if there's only one profile, but at multiple lines.

**Comment by ChunkyProgrammer** [comment]
For FreeTube we also import the subscriptions per line. 
Ref: https://github.com/FreeTubeApp/FreeTube/blob/2296e3a58d1895556bcec52b2162861140f295f8/src/renderer/components/data-settings/data-settings.js#L124

**Review by FireMasterK — APPROVED** [review]

File: src/components/ImportPage.vue [hunk 1]
```diff
@@ -105,10 +105,15 @@ export default {
...
-                    });
+                    const lines = text.split("\n");
+                    for (let i = 0; i < lines.length; i++) {
+                        if (lines[i] !== "") {
+                            const json = JSON.parse(lines[i]);
+                            json.subscriptions.forEach(item => {
+                                this.subscriptions.push(item.id);
+                            });
+                        }
+                    }
```
Comment (Bnyro) [inline src/components/ImportPage.vue:116]:
```suggestion
                    const lines = text.split("\n").filter(line => line !== "");
                    for (let line of lines) {
                        const json = JSON.parse(line);
                        json.subscriptions.forEach(item => {
                            this.subscriptions.push(item.id);
                        });
                    }
```

**Review by Bnyro — APPROVED** [review]

**Pull request merged**